### PR TITLE
Link back to translation

### PR DESCRIPTION
**Pull request by trymebytes**
Link these translations(see the screenshot below) to the translation page

Screenshot 2022-01-27 at 21 08 58

#### _This is how it now looks in the header and in the history tab;_
Screenshot 2022-01-28 at 09 31 40

#### _On the discussion tab, the translations in the comment footer have also been linked;_
Screenshot 2022-01-28 at 09 38 04

### CODE REVIEW

**Review by akirk — CHANGES_REQUESTED** [review]
Looks good at first glance but could you rebase before I do a full review? Currently this PR has a merge conflict.

**Comment by trymebytes** [comment]
> Looks good at first glance but could you rebase before I do a full review? Currently this PR has a merge conflict.

Done!

**Review by akirk — CHANGES_REQUESTED** [review]
There is something not quite right for waiting translations:
```
Waiting translation:
Notice: Undefined variable: translation_permalink in /Users/alex/Sites/localhost/wp/wp-content/plugins/gp-translation-tools/templates/original-permalink.php on line 88
```

Here is a waiting translation for me: https://glotpress.local/projects/wp/dev/2439646/zh-tw/default/

File: templates/original-permalink.php [hunk 1]
```diff
@@ -76,13 +85,13 @@
 					   ( '' == $e->translation_3 ) && ( '' == $e->translation_4 ) &&
 					   ( '' == $e->translation_5 ) ) :
```
Comment (akirk) [inline templates/original-permalink.php:86]: Would that work?
```suggestion
					   ( '' == $e->translation_5 ) ) :
					   $translation_permalink = GP_Route_Translation_Helpers::get_translation_permalink(
			$project,
			$locale_slug,
			$translation_set_slug,
			$original_id,
			$e->id
		);
```

**Comment by akirk** [comment]
I've added two things:

### Status background
Before:
Screenshot 2022-02-25 at 09 58 52
After:
Screenshot 2022-02-25 at 10 14 34

### Other locale links
Before:
Screenshot 2022-02-25 at 09 58 55

After:
Screenshot 2022-02-25 at 09 58 34

**Review by akirk — APPROVED** [review]
Ready to merge from me, @trymebytes could you also test my changes? Thanks!

**Comment by trymebytes** [comment]
> Ready to merge from me, @trymebytes could you also test my changes? Thanks!

Looks good to me! I'll go ahead and merge, thanks!

**Pull request merged**